ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/server/api/validation/ErrorFactoriesSpec.scala

```diff
@@ -531,6 +639,27 @@ class ErrorFactoriesSpec
     assertV2Error(error(errorFactoriesV2))(v2_code, v2_message, v2_details)
   }
 
+  private def assertVersionedStatus(
+      error: ErrorFactories => Status
+  )(
+      v1_code: Code,
```

Comment:
Please follow [Scala naming conventions](https://docs.scala-lang.org/style/naming-conventions.html) for arguments.